src/components/ImagePickerFormField.styles.js

```diff
@@ -35,8 +35,9 @@ export const styles = StyleSheet.create({
     fontSize: SIZES.md,
   },
   imageRemoveButton: {
-    flex: 2,
-    justifyContent: 'flex-end'
+    width: '89%',
+    justifyContent: 'flex-end',
+    marginBottom: -8,
```

Comment:
button was not near image so added styling for that and tested on ios and android, hopefully ok on physical devices